docs/documentation/publishing-on-heroku.md

```diff
@@ -43,22 +43,20 @@ You'll need to have [put your code on GitHub](/docs/github-desktop) to use this
 
 Your prototype will deploy automatically each time you push your code to GitHub (it takes a few minutes each time).
 
-## Set a username and password
+## Set a password
 
-We need to set a username and password or the Prototype Kit won’t run online. They don’t have to be complicated – it’s just to stop people accidentally coming across your prototype online and mistaking it for a real service.
+You need to set a password or the Prototype Kit will not run online. This password does not have to be complicated. It's just to stop people accidentally finding your prototype online and mistaking it for a real service.
 
-1. At the top click the **Settings** tab.
+1. At the top of the Heroku UI, click the **Settings** tab.
 
 2. Click **Reveal config vars**.
 
-3. In KEY put the word USERNAME
+3. In KEY, enter the word PASSWORD. Do not include the full stop.
```

Comment:
I know this is closer to style guide, but isn't it easier just to leave the full stop off as before?

Reply:
Sorry mate, got to have the full stop - they'll throw me out of the tech writers' club otherwise! The second sentence is overkill though, let's rip that out.

Reply:
Sorry, have to include it - they'll throw me out of the tech writers' club otherwise!